docs/gke.md

```diff
@@ -0,0 +1,73 @@
+# Google Kubernetes Engine (GKE) Deployment Guide
+
+Federation v2 can be deployed to and manage [GKE](https://cloud.google.com/kubernetes-engine/) clusters. Since
+Federation v2 requires Kubernetes v1.11 or greater, you must create
+[Alpha Clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alpha-clusters) and manually specify
```

Comment:
You are using `Alpha Clusters`, `Alpha clusters`, `alpha clusters`, is it possible to unify them?